onnx/backend/test/case/node/shape.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Shape(Base):
+
+    @staticmethod
+    def export():
+        node = onnx.helper.make_node(
+            'Shape',
+            inputs=['x'],
+            outputs=['y'],
+        )
+        x = np.random.randn(3, 4, 5).astype(np.float32)
```

Comment:
Could you add some cases only containing manually assigned inputs/outputs, something like the case here: https://github.com/onnx/onnx/blob/master/onnx/backend/test/case/node/sqrt.py#L23

That is easier for users to understand.

Reply:
Added